File: FE/AngularWeather/src/app/Models/Weather.ts
export class Weather{
    constructor(
        public city: string,
        public region: string,
        public country: string,
        public timeZone: string,
        public localTime: string,
        public temp_c: number,
        public windDirection: string,
        public condition: string,
        public conditionIcon: string,
        public wind_kph: number,
        public feelslike_c: number,
        public last_updated: string,
    ){}
}

File: BE/WeatherForecastApiToApi/Models/Weather.cs
namespace WeatherForecastApiToApi.Models
{
    public class Weather
    {
        public string City { get; set; }

        public string Region { get; set; }

        public string Country { get; set; }

        public string TimeZone { get; set; }

        public string LocalTime { get; set; }

        public double Temp_c { get; set; }

        public string WindDirection { get; set; }

        public string Condition { get; set; }

        public string ConditionIcon { get; set; }

        public double Wind_kph { get; set; }

        public double Feelslike_c { get; set; }

        public string Last_updated { get; set; }
    }
}
